setup.py

```diff
@@ -10,14 +10,14 @@
 
 setup(
     name = 'django-json-field',
-    version = '0.5.8',
+    version = '0.5.9',
     description = description,
     author = 'Derek Schaefer',
     author_email = 'derek.schaefer@gmail.com',
     url = 'https://github.com/derek-schaefer/django-json-field',
     long_description = long_description,
     packages = ['json_field'],
-    install_requires = ['django >= 1.2.7', 'python-dateutil', 'six >= 1.2.0'],
+    install_requires = ['django >= 3.2', 'python-dateutil', 'six >= 1.2.0'],
```

Comment:
Does this change actually introduce any incompatibilities with old Django versions? 
It looks like [re_path](https://docs.djangoproject.com/en/2.0/ref/urls/#re-path) was introduced in Django 2.0

```suggestion
    install_requires = ['django >= 2.2', 'python-dateutil', 'six >= 1.2.0'],
```
Since the 2.0 and 2.1 versions aren't LTS, they don't need to be supported. Not that we are going to do any testing against EOL versions.